tests/datadriven/test_variants.py

```
@@ -56,11 +57,65 @@ def getDataModelClass(self):
     def getProtocolClass(self):
         return protocol.GAVariantSet
 
-    def verifyVariantsEqual(self, gaVariants, pyvcfVariants):
+    def _verifyInfoEqual(self, gaObjectInfo, pyvcfInfo):
+        def _compareTwoFloats(a, b):
+            afloat = float(a)
+            bfloat = float(b)
+            return (abs(afloat - bfloat) <= 0.001 * bfloat)
+
+        def _compareGaValuePyvcfValue(gaValue, pyvcfValue):
+            if type(pyvcfValue) is str:
+                self.assertEqual(gaValue, pyvcfValue)
+            elif type(pyvcfValue) is int or type(value) is bool:
+                self.assertEqual(gaValue, str(pyvcfValue))
+            elif type(pyvcfValue) is float:
+                _compareTwoFloats(gaValue, pyvcfValue)
+            elif pyvcfValue is None:
+                self.assertEqual(gaValue, ".")
+            else:
+                print (gaValue, pyvcfValue)
```

We should raise an exception here.